.releaserc

```diff
@@ -0,0 +1,77 @@
+{
+  "branches": ["master"],
+  "repositoryUrl": "https://github.com/genesiscommunitysuccess/custom-elements-lsp",
+  "plugins": [
+    [
+      "@semantic-release/commit-analyzer",
+      {
+        "preset": "conventionalcommits",
+        "releaseRules": [
+          {
+            "type": "*!",
+            "release": "major"
+          }
+        ]
+      }
+    ],
+    [
+      "@semantic-release/release-notes-generator",
+      {
+        "preset": "conventionalcommits",
+        "presetConfig": {
+					"issuePrefixes": ["FUI-", "ISSUE-"],
+					"issueUrlFormat": "",
+					"linkReferences": false,
+        },
+        "releaseRules": [
+          {
+            "type": "*!",
+            "release": "major"
+          }
+        ]
+      }
+    ],
+    [
+      "@semantic-release/changelog",
+      {
+        "changelogFile": "CHANGELOG.md",
+        "changelogTitle": "# Changelog"
+      }
+    ],
+    [
+      "@google/semantic-release-replace-plugin",
+      {
+        "replacements": [
+          {
+            "files": ["package.json"],
+            "from": "\"version\": \".*\"",
+            "to": "\"version\": \"${nextRelease.version}\""
+          }
+        ]
+      }
+    ],
+    [
+      "@semantic-release/exec",
+      {
+        "prepareCmd": "npm version ${nextRelease.version} --no-git-tag-version",
+        "publishCmd": "npm publish"
+      }
+    ],
```

Comment:
We can use the NPM module here instead of manually calling these commands:
https://github.com/genesislcap/gpl-seed/blob/develop/.releaserc#L40

Reply:
Then the exec can also be dropped as a dep